sources/libcore/systemInformation.cpp

```diff
@@ -90,6 +125,16 @@ namespace cage
 		if (ret != 0)
 			CAGE_THROW_ERROR(systemError, "CallNtPowerInformation", ret);
 		return ppi[0].MaxMhz * 1000000;
+#elif defined CAGE_SYSTEM_LINUX
+		try {
+			holder<processHandle> prg = newProcess(string("cat /sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq"));
+			return prg->readLine().toUint32() * 1000;
+		} catch(exception e) {
```

Comment:
catch (const cage::exception &) // catch exceptions by const reference (also prefer to fully qualify that we are catching cage exception, not std exception)

Reply:
Added in new version of code

Reply:
Added in new version of code